### PR TITLE
Added native section in the docs

### PR DESCRIPTION
**Pull request by marcingrzejszczak**

### CODE REVIEW

File: BLOG.adoc [hunk 1]
```diff
@@ -359,6 +361,12 @@ class MyConfiguration {
 }
 ----
 
+=== Limitations
+
+The Spring Boot AutoConfiguration for WebMvc Observability is not yet there - that's why we needed to set things up manually. For more information check this https://github.com/spring-projects/spring-boot/issues/32538[issue].
```
Comment (Buzzardo) [inline BLOG.adoc:366]: Change to: 

The Spring Boot AutoConfiguration for WebMvc Observability is not yet ready. As a result, we need to set things up manually. For more information, see this https://github.com/spring-projects/spring-boot/issues/32538[issue].

File: BLOG.adoc [hunk 2]
```diff
@@ -359,6 +361,12 @@ class MyConfiguration {
...
+
+For the Spring Boot Exemplars AutoConfiguration to work properly we need to wait for https://github.com/spring-projects/spring-boot/pull/32399[this PR] and https://github.com/spring-projects/spring-boot/pull/32415[this PR] to be merged. Until then we need to create https://github.com/marcingrzejszczak/observability-boot-blog-post/blob/main/server/src/main/java/com/example/server/ExemplarsConfiguration.java[such configuration] manually.
```
Comment (Buzzardo) [inline BLOG.adoc:368]: Add commas after "work properly" and "Until then"

File: BLOG.adoc [hunk 3]
```diff
@@ -427,12 +435,105 @@ If you want to see the metrics from your Observation that you have manually crea
 
 image::https://raw.githubusercontent.com/marcingrzejszczak/observability-boot-blog-post/main/docs/src/main/asciidoc/img/my-observation.png[]
 
+== Running It All Together using Spring Native
+
+To better understand how Spring Boot supports Native please read https://spring.io/blog/2022/09/26/native-support-in-spring-boot-3-0-0-m5[this excellent blog post]. We will reuse that knowledge to run the previously created applications using Spring Native.
```
Comment (Buzzardo) [inline BLOG.adoc:440]: Add a comma after "supports Native" and remove "will"

File: BLOG.adoc [hunk 4]
```diff
@@ -427,12 +435,105 @@ If you want to see the metrics from your Observation that you have manually crea
...
+----
+
+Next, run the client side application.
+
+[indent=0]
+----
+$ ./client/target/client
+----
+
+You should get a similar output to this one
```
Comment (Buzzardo) [inline BLOG.adoc:476]: Change to: 

You should get output similar to this:

File: BLOG.adoc [hunk 5]
```diff
@@ -427,12 +435,105 @@ If you want to see the metrics from your Observation that you have manually crea
...
+2022-10-10T12:57:07.639+02:00  INFO [server,,] 81760 --- [nio-7654-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
+2022-10-10T12:57:07.639+02:00  INFO [server,,] 81760 --- [nio-7654-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
+2022-10-10T12:57:07.640+02:00  INFO [server,,] 81760 --- [nio-7654-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 1 ms
+2022-10-10T12:57:17.785+02:00  INFO [server,,] 81760 --- [nio-7654-exec-8] com.example.server.MyHandler             : Before running the observation for context [http.server.requests]
+2022-10-10T12:57:17.785+02:00  INFO [server,27c1113e4276c4173daec3675f536bf4,9affba5698490e2d] 81760 --- [nio-7654-exec-8] com.example.server.MyController          : Got a request
+2022-10-10T12:57:17.820+02:00  INFO [server,,] 81760 --- [nio-7654-exec-8] com.example.server.MyHandler             : After running the observation for context [http.server.requests]
+----
+
+You can check Grafana for metrics and traces. Read the limitations section below on why you won't find any pushed logs to Loki.
```
Comment (Buzzardo) [inline BLOG.adoc:521]: Turn "limitations section" into a link and remove "below" (which you won't need, thanks to the link).

File: BLOG.adoc [hunk 6]
```diff
@@ -427,12 +435,105 @@ If you want to see the metrics from your Observation that you have manually crea
...
+
+=== Native Support Limitations
+
+You will not see logs being pushed to Loki just yet. For more information look into https://github.com/spring-projects/spring-boot/issues/25847[this issue].
```
Comment (Buzzardo) [inline BLOG.adoc:525]: Add a comma after "For more information"

File: BLOG.adoc [hunk 7]
```diff
@@ -427,12 +435,105 @@ If you want to see the metrics from your Observation that you have manually crea
...
+
+On the client side we needed to provide the `reflect-config.js` configuration manually. For more information look into https://github.com/open-telemetry/opentelemetry-java/pull/4832[this PR].
```
Comment (Buzzardo) [inline BLOG.adoc:527]: Change to:

On the client side, we need to provide the `reflect-config.js` configuration manually. For more information, see https://github.com/open-telemetry/opentelemetry-java/pull/4832[this PR].

File: BLOG.adoc [hunk 8]
```diff
@@ -427,12 +435,105 @@ If you want to see the metrics from your Observation that you have manually crea
...
+
 = Summary
 
 In this blog post, we have managed to give you an introduction of the main concepts behind the Micrometer Observability API. We have also shown you how you can create observations by using the Observation API and annotations. You were also able to visualize the latency, see the correlated logs, and check the metrics that come from you Spring Boot applications.
 
+You were also able to observe your applications using native images via Spring Native.
```
Comment (Buzzardo) [inline BLOG.adoc:533]: Change to:

You could also able observe your applications by using native images with Spring Native.

File: BLOG.adoc [hunk 9]
```diff
@@ -427,12 +435,105 @@ If you want to see the metrics from your Observation that you have manually crea
...
+
 = Next Steps
 
 Based on community feedback, we will continue to improve our Observability story. We intend to go GA in November this year.
 
-This is an exciting time for us. We would again like to thank everyone who has already contributed and reported feedback, and we look forward to further feedback!
+This is an exciting time for us. We would again like to thank everyone who has already contributed and reported feedback, and we look forward to further feedback (don't hesitate to check out Spring Boot's latest snapshots)!
```
Comment (Buzzardo) [inline BLOG.adoc:539]: Change to:

This is an exciting time for us. We would again like to thank everyone who has already contributed and reported feedback, and we look forward to further feedback! Check out Spring Boot's latest snapshots!

File: docs/src/main/asciidoc/BLOG.adoc [hunk 10]
```diff
@@ -192,18 +194,117 @@ include::../../../../client/src/main/java/com/example/client/ClientApplication.j
 }
 ----
 
+=== Limitations
+
+The Spring Boot AutoConfiguration for WebMvc Observability is not yet there - that's why we needed to set things up manually. For more information check this https://github.com/spring-projects/spring-boot/issues/32538[issue].
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/BLOG.adoc:199]: Change to:

The Spring Boot AutoConfiguration for WebMvc Observability is not yet ready. As a result, we need to set things up manually. For more information, see this https://github.com/spring-projects/spring-boot/issues/32538%5Bissue%5D.

File: docs/src/main/asciidoc/BLOG.adoc [hunk 11]
```diff
@@ -192,18 +194,117 @@ include::../../../../client/src/main/java/com/example/client/ClientApplication.j
...
+
+For the Spring Boot Exemplars AutoConfiguration to work properly we need to wait for https://github.com/spring-projects/spring-boot/pull/32399[this PR] and https://github.com/spring-projects/spring-boot/pull/32415[this PR] to be merged. Until then we need to create https://github.com/marcingrzejszczak/observability-boot-blog-post/blob/main/server/src/main/java/com/example/server/ExemplarsConfiguration.java[such configuration] manually.
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/BLOG.adoc:201]: Add commas after "work properly" and "Until then"

File: docs/src/main/asciidoc/BLOG.adoc [hunk 12]
```diff
@@ -192,18 +194,117 @@ include::../../../../client/src/main/java/com/example/client/ClientApplication.j
...
+
 == Running It All Together
 
 We have prepared a Docker setup of the whole Observability infrastructure under https://github.com/marcingrzejszczak/observability-boot-blog-post[this link]. Follow these steps to run the infrastructure and both applications:
 
 include::../../../../RUNNING.adoc[]
 
+== Running It All Together using Spring Native
+
+To better understand how Spring Boot supports Native please read https://spring.io/blog/2022/09/26/native-support-in-spring-boot-3-0-0-m5[this excellent blog post]. We will reuse that knowledge to run the previously created applications using Spring Native.
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/BLOG.adoc:211]: Add a comma after "supports Native" and remove "will"

File: docs/src/main/asciidoc/BLOG.adoc [hunk 13]
```diff
@@ -192,18 +194,117 @@ include::../../../../client/src/main/java/com/example/client/ClientApplication.j
...
+=== Building
+
+To build the applications, you need GraalVM on your path. If you use `SDKMan`, invoke the following:
+
+[indent=0]
+----
+sdk install java 22.2.r17-nik
+----
+
+See also https://www.graalvm.org/java/quickstart/[GraalVM Quickstart]
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/BLOG.adoc:222]: Add a period at the end.

File: BLOG.adoc [hunk 14]
```diff
@@ -427,12 +435,105 @@ If you want to see the metrics from your Observation that you have manually crea
...
+=== Building
+
+To build the applications, you need GraalVM on your path. If you use `SDKMan`, invoke the following:
+
+[indent=0]
+----
+sdk install java 22.2.r17-nik
+----
+
+See also https://www.graalvm.org/java/quickstart/[GraalVM Quickstart]
```
Comment (Buzzardo) [inline BLOG.adoc:451]: Add a period at the end.

File: docs/src/main/asciidoc/BLOG.adoc [hunk 15]
```diff
@@ -192,18 +194,117 @@ include::../../../../client/src/main/java/com/example/client/ClientApplication.j
...
+----
+
+Next, run the client side application.
+
+[indent=0]
+----
+$ ./client/target/client
+----
+
+You should get a similar output to this one
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/BLOG.adoc:247]: Change to:

You should get output similar to this:

File: docs/src/main/asciidoc/BLOG.adoc [hunk 16]
```diff
@@ -192,18 +194,117 @@ include::../../../../client/src/main/java/com/example/client/ClientApplication.j
...
+2022-10-10T12:57:07.639+02:00  INFO [server,,] 81760 --- [nio-7654-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
+2022-10-10T12:57:07.639+02:00  INFO [server,,] 81760 --- [nio-7654-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
+2022-10-10T12:57:07.640+02:00  INFO [server,,] 81760 --- [nio-7654-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 1 ms
+2022-10-10T12:57:17.785+02:00  INFO [server,,] 81760 --- [nio-7654-exec-8] com.example.server.MyHandler             : Before running the observation for context [http.server.requests]
+2022-10-10T12:57:17.785+02:00  INFO [server,27c1113e4276c4173daec3675f536bf4,9affba5698490e2d] 81760 --- [nio-7654-exec-8] com.example.server.MyController          : Got a request
+2022-10-10T12:57:17.820+02:00  INFO [server,,] 81760 --- [nio-7654-exec-8] com.example.server.MyHandler             : After running the observation for context [http.server.requests]
+----
+
+You can check Grafana for metrics and traces. Read the limitations section below on why you won't find any pushed logs to Loki.
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/BLOG.adoc:292]: Turn "limitations section" into a link and remove "below" (which you won't need, thanks to the link).

File: docs/src/main/asciidoc/BLOG.adoc [hunk 17]
```diff
@@ -192,18 +194,117 @@ include::../../../../client/src/main/java/com/example/client/ClientApplication.j
...
+
+=== Native Support Limitations
+
+You will not see logs being pushed to Loki just yet. For more information look into https://github.com/spring-projects/spring-boot/issues/25847[this issue].
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/BLOG.adoc:296]: Add a comma after "For more information"

File: docs/src/main/asciidoc/BLOG.adoc [hunk 18]
```diff
@@ -192,18 +194,117 @@ include::../../../../client/src/main/java/com/example/client/ClientApplication.j
...
+
+On the client side we needed to provide the `reflect-config.js` configuration manually. For more information look into https://github.com/open-telemetry/opentelemetry-java/pull/4832[this PR].
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/BLOG.adoc:298]: Change to:

On the client side, we need to provide the reflect-config.js configuration manually. For more information, see https://github.com/open-telemetry/opentelemetry-java/pull/4832%5Bthis PR].

File: docs/src/main/asciidoc/BLOG.adoc [hunk 19]
```diff
@@ -192,18 +194,117 @@ include::../../../../client/src/main/java/com/example/client/ClientApplication.j
...
+
 = Summary
 
 In this blog post, we have managed to give you an introduction of the main concepts behind the Micrometer Observability API. We have also shown you how you can create observations by using the Observation API and annotations. You were also able to visualize the latency, see the correlated logs, and check the metrics that come from you Spring Boot applications.
 
+You were also able to observe your applications using native images via Spring Native.
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/BLOG.adoc:304]: Change to:

You could also able observe your applications by using native images with Spring Native.

File: docs/src/main/asciidoc/BLOG.adoc [hunk 20]
```diff
@@ -192,18 +194,117 @@ include::../../../../client/src/main/java/com/example/client/ClientApplication.j
...
+
 = Next Steps
 
 Based on community feedback, we will continue to improve our Observability story. We intend to go GA in November this year.
 
-This is an exciting time for us. We would again like to thank everyone who has already contributed and reported feedback, and we look forward to further feedback!
+This is an exciting time for us. We would again like to thank everyone who has already contributed and reported feedback, and we look forward to further feedback (don't hesitate to check out Spring Boot's latest snapshots)!
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/BLOG.adoc:310]: Change to:

This is an exciting time for us. We would again like to thank everyone who has already contributed and reported feedback, and we look forward to further feedback! Check out Spring Boot's latest snapshots!

**Review by Buzzardo — CHANGES_REQUESTED** [review]

**Comment by marcingrzejszczak** [comment]
Done via https://github.com/marcingrzejszczak/observability-boot-blog-post/commit/f5319252c6d891024fa1d42c42b38a2283b8a40e

**Pull request closed without merging**